### PR TITLE
chore: Update README

### PR DESCRIPTION
**Pull request by MicaiahReid**
- Add steps for deploying contract
 - Add steps for updating code in gallery ui
 - Add more context/explanations to steps
 - Separate installation steps from steps to use box
 - Add steps for uploading NFT metadata

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,16 +16,58 @@ The Filecoin box has the following requirements:
...
+ - (optionally) [Installing the Filecoin Network Explorer](#installing-the-filecoin-network-explorer), and
+ - [Installing the front-end gallery](#installing-the-front-end-gallery)
+
+#### Installing the Filecoin Box
+
+In a terminal window, start off by installing this box.
+
 ```bash
 $ truffle unbox filecoin
 $ npm install
```
Comment (its-everdred) [inline README.md]: Nit: Not something you did, but do you mind removing the "$ " in front of all the commands in this?  That will let users more easily copy/paste commands without deleting those characters from every line.  Example select:
image

File: README.md [hunk 2]
```diff
@@ -111,45 +160,69 @@ curl -X POST \
      http://localhost:7777/rpc/v0
 ```
 
-## Minting an NFT
+### Minting an NFT
 
-In the example below, we've already created a deal for the 3 assets (metadata, thumbnail, and the original asset respectively) that comprise our NFT. These are as follows, with their corresponding CIDs.
+In the example below, we've already created a deal for the 3 assets (metadata, thumbnail, and the original asset respectively) that comprise our NFT. These are as follows, with their corresponding Content Identifiers (aka CIDs).
```
Comment (its-everdred) [inline README.md]: Nice! Always a fan of defining acronyms and assuming no knowledge.  Technically, line 154 is the first usage, but either spot is probably fine.

**Review by its-everdred — APPROVED** [review]
Soooo much extra context. Looks great!

File: README.md (hunk 2 above)
Comment (MicaiahReid) [reply README.md]: I'll move it to line 154!

File: README.md (hunk 1 above)
Comment (MicaiahReid) [reply README.md]: agree, fixed!

**Pull request merged**